frontend/SideDish/SideDish/View/MainDiffableDataSource.swift

```diff
@@ -33,18 +32,19 @@ class MainDiffableDataSource {
         self.dataSource.apply(snapshot, animatingDifferences: true)
     }
     
-    func applySnapshot(menu:[MenuViewModel], section: sectionTitle) {
+    func applySnapshot(main: [MenuViewModel], soup: [MenuViewModel], side: [MenuViewModel]) {
         var snapshot = self.dataSource.snapshot()
-        snapshot.appendItems(menu, toSection: section)
-        
+        snapshot.appendItems(main, toSection: .main)
+        snapshot.appendItems(soup, toSection: .soup)
+        snapshot.appendItems(side, toSection: .side)
         self.dataSource.apply(snapshot, animatingDifferences: true)
     }
     
     func setupHeader(collectionView: UICollectionView, kind: String, indexPath: IndexPath) -> UICollectionReusableView? {
-        let header = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: "headerView", for: indexPath) as! HeaderView
+        let menuHeader = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: "menuHeaderView", for: indexPath) as! MenuHeaderView
```

Comment:
강제 언래핑 사용은 가급적 자제하세요!

```suggestion
        guard let menuHeader = collectionView.dequeueReusableSupplementaryView(ofKind: kind, withReuseIdentifier: "menuHeaderView", for: indexPath) as? MenuHeaderView else {
```